app/features/receive/cashu-receive-quote-service.ts

```diff
@@ -79,6 +85,65 @@ export class CashuReceiveQuoteService {
     return cashuReceiveQuote;
   }
 
+  /**
+   * Creates a new cashu receive quote and melts a token from a different mint to the destination account.
+   */
+  async createCashuTokenQuote({
```

Comment:
I am thinking about naming here and also if this belongs here.

While this technically is a lightning receive it's kind of different because we also pay it. Also so far when talking about this we have been calling this cross mint swap